test/controllers/concerns/current_user_test.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../../test_helper'
+
+SingleCov.covered! uncovered: 2
+
+describe "CurrentUser Controller" do
+  class CurrentUserTestController < ApplicationController
+    include CurrentUser
+
+    def whodunnit
+      render text: PaperTrail.whodunnit.to_s.dup
+    end
+
+    def change
+      Stage.find(params[:id]).update_attribute(:name, 'MUUUU')
+      head :ok
+    end
+  end
+
+  tests CurrentUserTestController
+  use_test_routes
+
+  as_a_viewer do
+    it "knows who did something" do
+      PaperTrail.with_whodunnit nil do
+        PaperTrail.whodunnit.must_equal nil
+        get :whodunnit, test_route: true
+        PaperTrail.whodunnit.must_equal nil
```

Comment:
I feel like this assertion could be a different `it` block. `it 'doesnt assign a user to something they didnt do' do ...`